docs/graphql/manual/guides/docker-networking.rst

```diff
@@ -0,0 +1,66 @@
+.. meta::
+   :description: Docker networking with Hasura
+   :keywords: hasura, docs, deployment, network, docker
+
+.. _docker_networking:
+
+Docker networking 
+=================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 2
+  :local:
+
+Introduction
+------------
+
+Sometimes you might want to connect Hasura with APIs that are either running outside of Docker or in a different Docker container.
```

Comment:
can we give examples of APIs here (auth webhook, event triggers, remote schemas)